lib/state/action-types.ts

```diff
@@ -101,7 +101,13 @@ export type SetUnsyncedNoteIds = Action<
   { noteIds: T.EntityId[] }
 >;
 export type ShowAllNotes = Action<'SHOW_ALL_NOTES'>;
-export type ShowDialog = Action<'SHOW_DIALOG', { dialog: T.DialogType }>;
+export type ShowDialog = Action<
+  'SHOW_DIALOG',
+  {
+    dialogName: T.DialogType['type'];
+    dialogData: object;
```

Comment:
generally speaking we can omit the prefix-as-namespace since the action type itself indicates we are talking about a dialog. in the previous code we had to have something there to indicate which one, and we probably could have used `name` but in the absence of a clear word for it, `dialog` communicated "which dialog"

here we can probably communicate just as effectively if we store `name` and `data`

Reply:
Sounds good and makes sense to me.